screens/stats/achievements/achievement.tsx

```diff
@@ -1,6 +1,47 @@
-// we will have to pass some props here
-const Achievement = () => {
-  return <div>THIS IS EXAMPLE ACHIVEMNT</div>;
+import { Card, Flex, Image, Progress, Text, Tooltip } from "@mantine/core";
+import { useMediaQuery } from "@mantine/hooks";
+
+interface AchievementProps {
+  achievement: {
+    defaultvalue: number;
+    displayName: string;
+    hidden: number;
+    description: string;
+    icon: string;
+    icongray: string;
+    globalPercent: number;
+  };
+}
+
+const Achievement = ({ achievement }: AchievementProps) => {
+  const isMobile = useMediaQuery("(max-width: 768px)");
+
+  return (
+    <Card my={5}>
+      <Flex justify={"space-between"} align={"center"} direction={"row"} mx={10}>
+        <Card.Section w={isMobile ? "80%" : 500}>
+          <Flex gap={5} direction={"column"}>
+            <Text fw={"bold"} size={isMobile ? "md" : "lg"}>
+              {achievement.displayName}
+            </Text>
+            <Text size={isMobile ? "xs" : "sm"}>{achievement.description}</Text>
+            <Tooltip label="">
+              <Progress
+                radius={"xs"}
+                size={isMobile ? "md" : "xl"}
+                value={achievement.globalPercent}
+                animate
+                label={`${achievement.globalPercent.toFixed(1)}%`}
+              />
+            </Tooltip>
+          </Flex>
+        </Card.Section>
+        <Card.Section>
+          <Image src={achievement.icon} alt="" height={isMobile ? 50 : 100}></Image>
```

Comment:
We can add alt `achievement.displayName-icon`, the SEO really likes to see alt on the images. Also if your element in React doesn't have any childs you can do it like this:
`<Image src={achievement.icon} alt="" height={isMobile ? 50 : 100} />`
instead of 
`<Image src={achievement.icon} alt="" height={isMobile ? 50 : 100}></Image>`